.github/workflows/release-sdk-to-candidate.yml

```diff
@@ -0,0 +1,67 @@
+name: Release ffmpeg-2204-sdk to candidate
+
+on:
+  # Run the workflow each time new commits are pushed to the candidate branch. 
+  push:
+    branches: [ "2204" ]
+    paths: 
+      - "ffmpeg-2204-sdk/**"
+  workflow_dispatch:
+    
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+  issues: write
+
+jobs:
+  get-architectures:
+    name: 🖥 Get snap architectures
+    runs-on: ubuntu-latest
+    outputs:
+      architectures: ${{ steps.get-architectures.outputs.architectures }}
+      architectures-list: ${{ steps.get-architectures.outputs.architectures-list }}
+    steps:
+      - name: 🖥 Get snap architectures
+        id: get-architectures
+        uses: snapcrafters/ci/get-architectures@main
+        with:
+          snapcraft-project-root: "ffmpeg-2204-sdk"
+
+  release-sdk:
+    name: 🚢 Release to latest/candidate
+    needs: get-architectures
+    runs-on: ubuntu-latest
+    environment: "2204 Branch"
+    strategy:
+      matrix:
+        architecture: ${{ fromJSON(needs.get-architectures.outputs.architectures-list) }}
+    steps:
+      - name: 🚢 Release to latest/candidate
+        uses: soumyaDghosh/ci/release-to-candidate@main
```

Comment:
```suggestion
        uses: snapcrafters/ci/release-to-candidate@main
```

Reply:
Waiting for the CI PR to merge before changing this.